jsonminify/index.d.ts

```diff
@@ -3,6 +3,6 @@
 // Definitions by: Dan Homola <https://github.com/no23reason>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function minify(json: string): string;
+declare const minify: (json: string) => string;
```

Comment:
The simplification you suggested @andy-ms turned out to break the CI (I apologise for committing it without thoroughly testing it). This version should be finally ok.